packages/core/admin/admin/src/pages/App/index.js

```diff
@@ -49,155 +69,172 @@ function App() {
-            // NOTE: needed because the interceptors of the fetchClient redirect to /login when receive a 401 and it would end up in an infinite loop when the user doesn't have a session.
-            validateStatus: (status) => status < 500,
-          });
-
-          setTelemetryProperties(properties);
-
-          try {
-            const event = 'didInitializeAdministration';
-            await post(
-              'https://analytics.strapi.io/api/v2/track',
-              {
-                // This event is anonymous
-                event,
-                userId: '',
-                deviceId,
-                eventPropeties: {},
-                userProperties: { environment: appInfo.currentEnvironment },
-                groupProperties: { ...properties, projectId: uuid },
-              },
-              {
-                headers: {
-                  'X-Strapi-Event': event,
-                },
-              }
-            );
-          } catch (e) {
-            // Silent.
-          }
-        }
-
-        setState({ isLoading: false, hasAdmin, uuid, deviceId });
-      } catch (err) {
-        toggleNotification({
-          type: 'warning',
-          message: { id: 'app.containers.App.notification.error.init' },
+    {
+      queryKey: 'telemetry-properties',
+      async queryFn() {
+        const {
+          data: { data },
+        } = await get(`/admin/telemetry-properties`, {
+          // NOTE: needed because the interceptors of the fetchClient redirect to /login when receive a
+          // 401 and it would end up in an infinite loop when the user doesn't have a session.
+          validateStatus: (status) => status < 500,
         });
-      }
-    };
 
-    getData();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [toggleNotification, updateProjectSettings]);
+        return data;
+      },
+
+      enabled: !!auth.getToken(),
+    },
+  ]);
+
+  React.useEffect(() => {
+    // If the renew token could not be fetched, logout the user
+    if (errorRenewToken) {
+      auth.clearAppStorage();
+      window.location.reload();
+    } else if (token) {
+      auth.updateToken(token);
+    }
+  }, [errorRenewToken, token]);
+
+  // Store the fetched project settings (e.g. logos)
+  // TODO: this should be moved to redux
+  React.useEffect(() => {
+    if (!isLoadingInit && initData) {
+      updateProjectSettings({
+        menuLogo: prefixFileUrlWithBackendUrl(initData.menuLogo),
+        authLogo: prefixFileUrlWithBackendUrl(initData.authLogo),
+      });
+
+      // TODO: this should be stored in redux
+      setState((prev) => ({
+        ...prev,
+        hasAdmin: initData.hasAdmin,
+        uuid: initData.uuid,
+      }));
+    }
+  }, [initData, isLoadingInit, updateProjectSettings]);
+
+  // we can't use useTracking here, because `App` is not wrapped in the tracking provider
+  // context. This shouldn't use `useFetchClient`, because it does not talk to the admin
+  // API
```

Comment:
why are we not wrapping it then? :)

Reply:
Because the `AppInfoProvider` is only part of the authenticated app, rather than both: authenticated and un-authenticated. So we can't move it a level up, but the tracking needs to happen for both.